easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.0-foss-2016b.eb

```diff
@@ -0,0 +1,62 @@
+name = 'QuantumESPRESSO'
+version = '6.0'
+
+homepage = 'http://www.pwscf.org/'
+description = """Quantum ESPRESSO  is an integrated suite of computer codes
+ for electronic-structure calculations and materials modeling at the nanoscale.
+ It is based on density-functional theory, plane waves, and pseudopotentials
+  (both norm-conserving and ultrasoft)."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+# major part of this list was determined from espresso/install/plugins_list
+sources = [
+    'qe-%(version)s.tar.gz',
+    'wannier90-1.2.tar.gz',
+    'qe-gipaw-6.0.tar.gz',
+    # must be downloaded manually from
+    # http://qe-forge.org/gf/project/q-e/scmsvn/?action=browse&path=%2F%2Acheckout%2A%2Ftags%2FQE-5.2.1%2Fespresso%2Farchive%2Fplumed-1.3-qe.tar.gz&revision=11758
+    # gets updated without changes to filename, cfr. http://qe-forge.org/pipermail/q-e-commits/2015-June/007359.html
+    # 'plumed-1.3-qe-r11758.tar.gz',
+    'want-2.5.1-base.tar.gz',
+    'yambo-3.4.2.tgz',
+]
+missing_sources = [
+    'sax-2.0.3.tar.gz',  # nowhere to be found
+]
+source_urls = [
+    'http://files.qe-forge.org/index.php?file=',  # all sources, except espresso*.tar.gz
+    'http://qe-forge.org/gf/download/frsrelease/224/1044/',  # qe-6.0.tar.gz
+    'http://qe-forge.org/gf/download/frsrelease/153/618/',  # want-2.5.1-base.tar.gz
+    'http://qe-forge.org/gf/download/frsrelease/208/932/',  # yambo-3.4.2.tgz
+]
+
+patches = [
+    'QuantumESPRESSO-6.0_yambo-fixes-foss-2016b.patch',
+    'want-2.5.1-gfortran.patch'
+]
+
+# source checksums
+checksums = [
+    'e42aeeffadf7951542d8561a6b4a3390',  # qe-6.0.tar.gz
+    'a1e9611b9a82941c23426028d112186e',  # wannier90-1.2.tar.gz
+    'b11a8b75109eb58fd97a1c6f7cf20e31',  # qe-gipaw-6.0.tar.gz
+    # 'f094031c6d13a0e00022daf4d7c847c7',  # plumed-1.3-qe-r11758.tar.gz
+    'ac365daebbe380bf4019235eacf71079',  # want-2.5.1-base.tar.gz
+    'f0820a0f42dfc81ce3811af647bed777',  # yambo-3.4.2.tgz
+]
+
+# dependencies are required by yambo
+dependencies = [
+    ('netCDF-Fortran', '4.4.4'),
+    ('libxc', '2.2.3'),
+]
```

Comment:
@ocaisa hmm, these dependencies are not there in the QE 6.0 easyconfig using `intel/2016b` that was added in #3647, so we should include them there too? Are you up for doing that (I can take care of retesting)?